src/lib/annotations/Annotator.js

```diff
@@ -89,7 +90,6 @@ class Annotator extends EventEmitter {
      * @return {void}
      */
     init() {
-        this.annotatedElement = this.getAnnotatedEl(this.container);
```

Comment:
Maybe throw an error here if a developer forgets to override getAnnotatotedEl() in their viewer specific annotator